examples/azure-tf-example/porter.yaml

```diff
@@ -7,11 +7,10 @@ mixins:
 ## This section defines the metadata that defines the bundle. This includes the name of the generated
 ## invocation image as well as the tag that will be applied to the bundle when it is published.
 
-name: porter-terraform-example
+name: porter-azure-terraform
```

Comment:
Same let's drop the porter-* prefix from the bundle name too, in addition to the bundle tag.